src/QsCompiler/Transformations/MonomorphizationTransformation.cs

```diff
@@ -10,15 +10,16 @@
 using Microsoft.Quantum.QsCompiler.SyntaxTokens;
 using Microsoft.Quantum.QsCompiler.SyntaxTree;
 using Microsoft.Quantum.QsCompiler.Transformations.Core;
+using Microsoft.Quantum.QsCompiler.Transformations.SearchAndReplace;
 
 
-namespace Microsoft.Quantum.QsCompiler.Transformations.MonomorphizationTransformation
+namespace Microsoft.Quantum.QsCompiler.Transformations.Monomorphization
```

Comment:
Same suggestion about renaming file to Monomorphization.cs.

Reply:
I'll resolve the rest of the comments, leaving one open until we decide.